k8s-ci/test

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -euo pipefail
+cd "$(dirname "$0")"
+. ./utils.sh
+
+# Set environment variables
+export UNIQUE_TEST_ID
+UNIQUE_TEST_ID="$(uuidgen | tr "[:upper:]" "[:lower:]" | head -c 12 | tr -d -)"
+export SECRETLESS_IMAGE="${DOCKER_REGISTRY_PATH}/secretless-broker:${UNIQUE_TEST_ID}"
```

Comment:
Would it be useful to build and tag the secretless-broker image here? I had to manually build it to get the tests to pass.

Reply:
This is normally run preceded by a build stage in Jenkins so it assumes the image has been built already. I'm wondering if there's a better way to make that relationship explicit. Perhaps a check before the script runs that complains if there's no such image

Reply:
Ah of course, it runs in an earlier pipeline stage!